File: auth-authorization-server-api/src/main/java/com/cartobucket/auth/authorization/server/dto/JWKS.java
package com.cartobucket.auth.authorization.server.dto;

import jakarta.json.bind.annotation.JsonbProperty;
import jakarta.json.bind.annotation.JsonbNillable;
import java.util.List;

@JsonbNillable(false)
public class JWKS {
    
    @JsonbProperty("keys")
    private List<JWK> keys;
    
    public List<JWK> getKeys() {
        return keys;
    }
    
    public void setKeys(List<JWK> keys) {
        this.keys = keys;
    }
}

File: auth-authorization-server-api/src/main/kotlin/com/cartobucket/auth/authorization/server/dto/JWKS.kt
// (C)2024
package com.cartobucket.auth.authorization.server.dto

import jakarta.json.bind.annotation.JsonbNillable
import jakarta.json.bind.annotation.JsonbProperty

@JsonbNillable(false)
data class JWKS(
    @field:JsonbProperty("keys")
    val keys: List<JWK>? = null,
)
